File: backend/voting-app/src/main/java/com/tu/votingapp/dto/response/elections/PartyResponseDTO.java
package com.tu.votingapp.dto.response.elections;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class PartyResponseDTO {
    private Long id;
    private String name;
    private String abbreviation;
    private String logoUrl;
    private String leaderName;
    private List<CandidateResponseDTO> candidates;
}

File: frontend/android/app/src/main/java/com/example/android/dto/response/elections/PartyResponseDTO.kt
package com.example.android.dto.response.elections

import kotlinx.serialization.Serializable

@Serializable
data class PartyResponseDTO(
    val id: Long,
    val name: String,
    val abbreviation: String,
    val logoUrl: String,
    val leaderName: String,
    val candidates: List<CandidateResponseDTO>
)